### ISSUE TITLE
Production ready examples

### ISSUE THREAD

**Issue by notrab** [prisma/nexus#146]
Hey

Would anyone be interested in helping out adding some build scripts for the examples? I noticed they mostly use ts-node-dev which isn’t ideal for deployment.

**Comment by tgriesser** [comment]
> Would anyone be interested in helping out adding some build scripts for the examples? I noticed they mostly use ts-node-dev which isn’t ideal for deployment.

Yes, I think we should definitely add/link to more examples of real world apps. One thing I wanted to point out, is that `node -r ts-node/register/transpile-only app.ts` is actually a fairly reasonable option depending on your environment and needs, it eliminates the confusion of the build step and the potential of things falling out of sync in development, which can be confusing to debug. 

Obviously, YMMV but this is how how all of our Node apps are currently running at Cypress.io (after things are properly type-checked in CI prior to deploy) and it hasn't been an issue, aside from a slightly inflated memory footprint at startup, but overall the overhead of transpiling on the fly has appeared to be negligible.

**Comment by hassanshk866** [comment]
Use a dictionary to store information about a person you know. Store their first name, last name, age, and the city in which they live. You should have keys such as first_name, last_name, age, and city. Print each piece of information stored in your dictionary. Add a new key value pair about qualification then update the qualification value to high academic level then delete it.


plz HOW CAN MAKE A PROGRAM
